satellite/gracefulexit/endpoint_test.go

```diff
@@ -495,13 +637,12 @@ func TestFailureUnknownError(t *testing.T) {
 			require.FailNow(t, "should not reach this case: %#v", m)
 		}
 
-		// TODO uncomment once progress reflects updated success and fail counts
 		// check that the exit has completed and we have the correct transferred/failed values
-		// progress, err := satellite.DB.GracefulExit().GetProgress(ctx, exitingNode.ID())
-		// require.NoError(t, err)
-		//
-		// require.Equal(t, int64(0), progress.PiecesTransferred, tt.name)
-		// require.Equal(t, int64(1), progress.PiecesFailed, tt.name)
+		progress, err := satellite.DB.GracefulExit().GetProgress(ctx, exitingNode.ID())
+		require.NoError(t, err)
+
+		require.Equal(t, int64(0), progress.PiecesTransferred)
+		require.Equal(t, int64(0), progress.PiecesFailed)
```

Comment:
hmm why is pieces failed 0?

Reply:
Because It hasn't  failed the same piece above the per piece failure threshold